packages/cron/README.md

```diff
@@ -1,73 +1,103 @@
+
+[![GitHub](https://img.shields.io/github/license/Kings-World/sapphire-plugins)](https://github.com/Kings-World/sapphire-plugins/blob/main/LICENSE.md)
+[![npm bundle size](https://pkg-size.dev/badge/bundle/83411)](https://pkg-size.dev/@kingsworld/plugin-cron)
+[![npm](https://img.shields.io/npm/v/@kingsworld/plugin-cron?color=crimson&logo=npm&style=flat-square)](https://www.npmjs.com/package/@kingsworld/plugin-cron)
+
+</div>
+
+## Description
+
+This plugin adds support for cron tasks to the Sapphire framework. It uses the [cron](ttps://www.npmjs.com/package/@kingsworld/plugin-cron) package to create and manage cron tasks.
+
+## Features
+
+-   Full TypeScript support
+-   Includes ESM entrypoint
 
 ## Installation
 
+`@kingsworld/plugin-cron` depends on the following packages. Be sure to install these along with this package!
+
+-   [`@sapphire/framework`](https://www.npmjs.com/package/@sapphire/framework)
+
+You can use the following command to install this package along with `cron`, or replace `npm install` with your package manager of choice.
+
 ```sh
-yarn add @kingsworld/plugin-cron @sapphire/framework
+npm install @kingsworld/plugin-cron @sapphire/framework
 ```
 
 ## Usage
 
-Make sure to register the plugin before creating the client
+This registers the necessary options and methods in the Sapphire client to be able to use the schedule plugin.
 
 ```ts
+// Main bot file
+// Be sure to register the plugin before instantiating the client.
 import '@kingsworld/plugin-cron/register';
 ```
 
-If you want to set the default cron job timezone for all your cron jobs, you can do so within the client options. A list of TZ identifiers can be found on [Wikipedia](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
+Then, you can configure the plugin in the configuration options in your SapphireClient extension class or initializer. This will either be located in your new SapphireClient constructor call, or super in your constructor method if you use an extension class.
 
 ```ts
-new SapphireClient({
-	...otherClientOptions,
+const options = {
+	...otherClientOptionsGoHere,
 	cron: {
-		// the cron object is optional
-		defaultTimezone: 'Europe/London' // the cron package defaults to UTC
+		/**
+		 * Whether to disable Sentry cron monitoring
+		 * @default true
+		 */
+		disableSentry: false,
+		/**
+		 * The timezone to use for the cron tasks
+		 * @default 'UTC'
+		 */
+		timeZone: 'Europe/London' // set the timezone
 	}
-});
+};
 ```
 
-However, if you would like to do so for a single task, you can do so in the cron task options
+-   The `disableSentry` option is used to disable [Sentry's cron monitoring](https://docs.sentry.io/product/crons/). If you do not use Sentry, you can safely ignore this option. If you use Sentry, you can set this to `false` to enable cron monitoring.
```

Comment:
> If you use Sentry, you can set this to `false` to enable cron monitoring.

Currently, this is written in a way that in my opinion, makes using Sentry opt-in with the `disableSentry` property.

The initial opt-in is having `@sentry/node` installed and then if you want to opt-out in certain situations, the `disableSentry` property can be used. I hope this makes sense.